src/Hedgehog/LinqSupport.fs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Hedgehog
+
+open System
+open System.Runtime.CompilerServices
+
+open Hedgehog
+
+[<Extension>]
+module GenLinqSupport =
+
+    // Support for `where`:
+    [<Extension>]
+    [<CompiledName("Where")>]
```

Comment:
Nitpick: This can also be expressed as `[<CompiledName "Where>]` without parenthesis, but I can go ahead and do this after we merge the PR 😅

Reply:
That looks a bit weird to me 👽

Reply:
Ugh, nevermind then 😅

Reply:
happy to change it though, but I haven't seen other F# code doing that... 🙂

Reply:
No, don't let this small thing slow you down 😃 

---

It's just that, the closest to our existing code style would really be

```f#
[<CompiledName ("Where")>] // Notice the space.  But then, the space isn't needed.
```